lib/migrations/readers/index.ts

```diff
@@ -0,0 +1,2 @@
+export { StorageReader2_0_0 } from './reader_2_0_0';
+export { StorageReader1_0_0 } from './reader_1_0_0';
```

Comment:
You might want to set `"files.insertFinalNewline": true` in your vscode config :)

Reply:
I actually don't like a final empty line 😳 Is this illegal?

Reply:
It's mostly about compatibility with older unix tools I believe https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file